### DISCUSSION TITLE
How to set target_bpp?

### DISCUSSION THREAD

**Discussion by using369** [tensorflow/compression#134] [Q&A]
Hi! I want to set the target bpp to encode an image by the  following command:
     `python tfci.py compress --target_bpp 0.5 --bpp_strict <input_file>`
However I got the following error:
      usage: tfci.py compress [-h] [--helpfull] [--target_bpp TARGET_BPP]
                        [--bpp_strict]
                        model input_file [output_file]
     tfci.py compress: error: the following arguments are required: input_file

I find it regarded <input_file> as <model>，because I did not specify the model identifier. But the help of parameter 'target_bpp' is shown below:
  "Target bits per pixel. If provided, a binary search is used to try  to match the given bpp as close as possible. In this case, don't            
    specify the index at the end of the model identifier. It will be  automatically determined."

I feel confused about  the parameter "--target_bpp". I want to know how to use it correctly. 
Many thanks!

**Comment by jonaballe** [comment]
Hi,

the online help basically means to say: specify `b2018-gdn-128` instead of `b2018-gdn-128-1`, for example (without the number at the end).

For example:
```
python tfci.py compress --target_bpp 0.5 --bpp_strict b2018-gdn-128 <input_file>
```

Hope this helps!
